### ISSUE TITLE
Get the image approved as  an "official Docker image"

### ISSUE THREAD

**Issue by potiuk** [apache/airflow#10107]
https://docs.docker.com/docker-hub/official_images/

Depends on #11740 #35021 and #34243

**Comment by eladkal** [comment]
@potiuk do we still have a task here?

**Comment by potiuk** [comment]
Yes. Part of it is signing the image and part of it automatically scanning the image (likely with Scout). 

Added #35021 and we already had #34243 covering both - I will be working on them soon as part of our security improvements.
